test-prow-e2e.sh

```diff
@@ -14,6 +21,4 @@ source ./contrib/oc-environment.sh
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_06-rh-operators.configmap.yaml
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_09-rh-operators.catalogsource.yaml
 
-./test-gui.sh e2e
-
-cp -rv ./frontend/gui_test_screenshots "${ARTIFACT_DIR}/gui_test_screenshots"
+. ./test-gui.sh e2e
```

Comment:
Is there a reason this needs to be `. ./test-gui.sh e2e` instead of just `./test-guit.sh e2e`?

Reply:
This will run the script and not exit the entire script if the test-gui.sh sub script returns an error. The trap should handle the error case I believe....I can remove this in the test pr and check the outcome.

Reply:
I'd think the trap would still work if we run it directly. Can we test?

Reply:
Yeah the test is running.